lib/compiler/defaults/swc-defaults.ts

```diff
@@ -25,8 +29,8 @@ export const swcDefaultsFactory = (tsOptions: ts.CompilerOptions) => {
       swcrc: true,
     },
     cliOptions: {
-      outDir: 'dist',
-      filenames: ['src'],
+      outDir: tsOptions.outDir ?? 'dist',
+      filenames: [configuration.sourceRoot ?? 'src'],
```

Comment:
"sourceRoot" is dedicated to schematics - it's not supposed to control builder's behavior. I think we should just expose the "swcCliOptions" prop (under the `compilerOptions` in `nest-cli.json`) that would let everyone override the default options - if needed.

Would you like to update the PR?

Reply:
Ok, I will update him, but this change will involve a schema change in `nest-cli.json`, also, the default outDir is set to `outDir` in tsconfig, which is more in line with configuration expectations.